documentation/docs/scenarios/CAP_Scenario.md

```diff
@@ -0,0 +1,65 @@
+# Build and Deploy Applications with Jenkins and the SAP Cloud Application Programming Model
```

Comment:
But how do customers know that this is a scenario that involves Jenkins when searching for it? For the findability, we should include Jenkins in the heading - otherwise, the difference between our scenario and the programming model itself isn't obvious. 

What do you think?

Reply:
Your arguments are valid, lets keep the Jenkins reference then